rest/model/admin.go

```diff
@@ -439,6 +450,30 @@ func (a *APIAmboyRetryConfig) ToService() (interface{}, error) {
 	}, nil
 }
 
+// APIAmboyNamedQueueConfig is the model for named Amboy queue settings.
+type APIAmboyNamedQueueConfig struct {
+	Name               *string `json:"name"`
```

Comment:
For my own understanding: What is the point of having this secondary struct, that only differs from `AmboyNamedQueueConfig` in that it has a string pointer instead of a string?

Reply:
The API-level struct is required to separate the REST request/response data model from the DB data model since they're different layers of abstraction.

Passing the named queues as a slice with the name as a field instead of using a map is just simpler for JSON payloads. It's not that desirable to use a slice where a map is more appropriate, but it makes writing the JS easier and it's not a major issue since admins are the only ones using it.